src/Codeception/Module/Yii2.php

```diff
@@ -868,29 +845,25 @@ public function _backupSession(): array
         return [
             'clientContext' => $this->getClient()->getContext(),
             'headers' => $this->headers,
-            'cookie' => isset($_COOKIE) ? $_COOKIE : [],
-            'session' => isset($_SESSION) ? $_SESSION : [],
+            'cookie' => $_COOKIE,
+            'session' => $_SESSION,
```

Comment:
Are we sure these global variables are set?

Reply:
[PHPStan does not know ](https://github.com/phpstan/phpstan/issues/9953)

I think it's a bug in phpstan. I'd just ignore the phpstan message tbh.